h2o-algos/src/test/java/hex/tree/SharedTreeTest.java

```diff
@@ -142,4 +143,29 @@ private TestFrameBuilder twoVecFrameBuilder() {
           .withDataForCol(1, ar("A", "B", "A", "B", "A", "B"));
   }
 
+  /**
+   * PUBDEV-8276 - 
+   */
+  @Test(expected = H2OModelBuilderIllegalArgumentException.class)
```

Comment:
I think the test has to validate what error is actually thrown

or I can go and accidentally delete a line from the test - eg. line 153 (with column names) and the test will start failing for a different reason => it will not be testing anything anymore

Reply:
Fixed.